Automattic/jetpack
# Testing: stop running e2e tests on master in CircleCI

This will be done in a separate project periodically rather than on every push to master

The results have been inconsistent and noisy and if we isolate the tests and run them less frequently this should reduce the impact on Jurassic.ninja

**Testing Instructions**

Just make sure CircleCI builds are no longer kicked off after this change
No functional changes in this PR
